common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1754,32 +1765,66 @@ object CelebornConf extends Logging {
+      .categories("worker")
+      .doc("Max ratio of sort memory.")
+      .version("0.2.0")
+      .doubleConf
+      .checkValue(v => v >= 0.0 && v <= 1.0, "should be in [0.0, 1.0].")
+      .createWithDefault(0.1)
 
-  def workerPausePushDataRatio(conf: CelebornConf): Double = {
-    conf.getDouble("rss.pause.pushdata.memory.ratio", 0.85)
-  }
+  val WORKER_DIRECT_MEMORY_RATIO_PAUSE_RECEIVE: ConfigEntry[Double] =
+    buildConf("celeborn.worker.directMemoryRatioToPauseReceive")
+      .withAlternative("rss.pause.pushdata.memory.ratio")
+      .categories("worker")
+      .doc("If direct memory usage reaches this limit, worker will stop to receive from executor.")
+      .version("0.2.0")
+      .doubleConf
+      .checkValue(v => v >= 0.0 && v <= 1.0, "should be in [0.0, 1.0].")
+      .createWithDefault(0.85)
 
-  def workerPauseRepcaliteRatio(conf: CelebornConf): Double = {
-    conf.getDouble("rss.pause.replicate.memory.ratio", 0.95)
-  }
+  val WORKER_DIRECT_MEMORY_RATIO_PAUSE_REPLICATE: ConfigEntry[Double] =
+    buildConf("celeborn.worker.directMemoryRatioToPauseReplicate")
+      .withAlternative("rss.pause.replicate.memory.ratio")
+      .categories("worker")
+      .doc("If direct memory usage reaches this limit, worker will stop to receive from executor and other worker.")
+      .version("0.2.0")
+      .doubleConf
+      .checkValue(v => v >= 0.0 && v <= 1.0, "should be in [0.0, 1.0].")
+      .createWithDefault(0.95)
 
-  def workerResumeRatio(conf: CelebornConf): Double = {
-    conf.getDouble("rss.resume.memory.ratio", 0.5)
-  }
+  val WORKER_DIRECT_MEMORY_RATIO_RESUME: ConfigEntry[Double] =
+    buildConf("celeborn.worker.directMemoryRatioToResume")
+      .withAlternative("rss.resume.memory.ratio")
+      .categories("worker")
+      .doc("If direct memory usage is less than this limit, worker will resume.")
+      .version("0.2.0")
+      .doubleConf
+      .checkValue(v => v >= 0.0 && v <= 1.0, "should be in [0.0, 1.0].")
+      .createWithDefault(0.5)
 
-  def initialReserveSingleSortMemory(conf: CelebornConf): Long = {
-    conf.getSizeAsBytes("rss.worker.initialReserveSingleSortMemory", "1mb")
-  }
+  val WORKER_DIRECT_MEMORY_CHECK_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.worker.memory.checkInterval")
+      .withAlternative("rss.worker.memory.check.interval")
+      .categories("worker")
+      .doc("Worker direct memory checking interval.")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("10ms")
 
-  def workerDirectMemoryPressureCheckIntervalMs(conf: CelebornConf): Int = {
-    conf.getInt("rss.worker.memory.check.interval", 10)
-  }
+  val WORKER_DIRECT_MEMORY_REPORT_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.worker.memory.reportInterval")
+      .withAlternative("rss.worker.memory.report.interval")
+      .categories("worker")
+      .doc("Worker direct memory tracker reporting to log interval.")
```

Comment:
`Worker direct memory tracker reporting interval.`

Reply:
Have discussed with pan cheng about this doc before, https://github.com/alibaba/celeborn/pull/831#discussion_r1004211709 ,but current doc seems also a bit weird, will change it.